### DISCUSSION TITLE
Please add command line keys

### DISCUSSION THREAD

**Discussion by SergS78** [xM4ddy/OFGB#16] [Ideas]
For example: `OFGB-Deps.exe /all /close /norestart`
or simply: `/s`
I need to run in silent mode and close the application witthout restart windows

**Comment by xM4ddy** [comment]
> For example: `OFGB-Deps.exe /all /close /norestart` or simply: `/s` I need to run in silent mode and close the application witthout restart windows

Hey please make an issue with this suggestion instead of a discussion
